File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/commands/CancelSeatReservation.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class CancelSeatReservation : Command<Guid>
    {
        public Guid ReservationId { get; set; }

        public CancelSeatReservation() { }
        public CancelSeatReservation(Guid conferenceId, Guid reservationId) : base(conferenceId)
        {
            ReservationId = reservationId;
        }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/CancelSeatReservation.java
package com.microsoft.conference.common.management.commands;

import org.enodeframework.commanding.Command;

public class CancelSeatReservation extends Command<String> {
    public String reservationId;

    public CancelSeatReservation() {
    }

    public CancelSeatReservation(String conferenceId, String reservationId) {
        super(conferenceId);
        this.reservationId = reservationId;
    }
}
